e2e/pages/strategy.spec.ts

```diff
@@ -520,7 +520,7 @@ const testCases: CreateStrategyTestCase[] = [
       create: {
         buy: {
           min: '0.3',
-          max: '0.545454',
```

Comment:
why would the numbers change?

Reply:
Because SDK doesn't use `toFixed` anymore.